### PR TITLE
[no ticket][risk=no] Update Python kernel name caused by Jupyter upgrade

### PR DESCRIPTION
**Pull request by yonghaoy**
https://user-images.githubusercontent.com/6351731/238725508-2384746b-3d12-4972-9e58-852aa6495ff1.png

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: e2e/tests/notebook/notebook-create-python.spec.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('Python Kernel Notebook Test', () => {
 
     // Verify kernel name.
     const kernelName = await notebook.getKernelName();
-    expect(kernelName).toBe('Python 3');
+    expect(kernelName).toBe('Python 3 (ipykernel)');
```
Comment (jmthibault79) [inline e2e/tests/notebook/notebook-create-python.spec.ts:55]: a const would be nice to have here - for R too

Comment (yonghaoy) [reply e2e/tests/notebook/notebook-create-python.spec.ts:55]: Just like other expected values, why a const be useful for once time use local variable?

Comment (jmthibault79) [reply e2e/tests/notebook/notebook-create-python.spec.ts:55]: Because it is not single-use.  It is updated twice in this PR.

Comment (yonghaoy) [reply e2e/tests/notebook/notebook-create-python.spec.ts:55]: oh you suggest to create a const and shared by both places? Any suggestions about where to put it and how to use that?

**Review by NehaBroad — APPROVED** [review]

**Review by jmthibault79 — APPROVED** [review]

**Pull request merged**